Gemfile.lock

```diff
@@ -1,3 +1,22 @@
+GIT
+  remote: https://github.com/wordpress-mobile/release-toolkit
+  revision: cb939d6f00ed0483eb4baf107361b7281bf9feac
```

Comment:
Since I approved [the release-toolkit PR](https://github.com/wordpress-mobile/release-toolkit/pull/258), feel free to merge it, then generate a new release-tookit version – see paaHJt-2d8-p2 for instructions and point your `fastlane/Pluginfile` to it now.

If you are still waiting for Oguz to review the PR before you feel confident in making a new release-toolkit release, I'm ok to merge this PR while still pointing on a toolkit branch for now, and we can do a subsequent PR to update this once the new toolkit release lands, but I feel we're ready to make that release now tho so why wait 😉